CHANGES.rst

```diff
@@ -4,6 +4,9 @@ Products.CMFCore Changelog
 2.4.0b6 (unreleased)
 --------------------
 
+- Check for `__self__` as well as `im_self` for Python 3 compatibility
```

Comment:
Replacing `im_self` with `__self__` should be already compatible with Python >= 2.7.

Reply:
Oh really, even better. So in Python 2.7 `im_self` is alias to `__self__`?

Reply:
See https://docs.python.org/2/reference/datamodel.html: 

> Changed in version 2.6: For Python 3 forward-compatibility, im_func is also available as __func__, and im_self as __self__.

Apparently this alias was added in Python 2.6 #TIL